wherehows-web/app/components/hotkeys/global-hotkeys.ts

```diff
@@ -56,11 +56,27 @@ export default class GlobalHotkeys extends Component {
    * Method for handling the global keyup.
    * @param {KeyboardEvent} e - KeyboardEvent triggered by user input
    */
-  keyUp(e: KeyboardEvent) {
+  onKeyUp(e: KeyboardEvent) {
     const target = <HTMLElement>e.target;
 
     if (this.isEligibleTarget(target)) {
       get(this, 'hotKeys').applyKeyMapping(e.keyCode);
     }
   }
+
+  didInsertElement() {
+    const body = document.getElementsByClassName('ember-application')[0];
```

Comment:
You could always use the `querySelector` instead of getting the first element of the getElementsByClassName. `querySelector` will always return 1 result only.

```js
const body = document.querySelector('.ember-application');
```